merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/MissionModelParser.java

```diff
@@ -104,7 +105,11 @@ private TypeElement getMissionModelModel(final PackageElement missionModelElemen
           attribute);
     }
 
-    return Optional.of((TypeElement) ((DeclaredType) attribute.getValue()).asElement());
+    final var declaration = (TypeElement) ((DeclaredType) attribute.getValue()).asElement();
+    final var name = declaration.getSimpleName().toString();
+    final var parameters = getParameters(declaration);
+    // TODO validation list is empty, not parsing those yet
+    return Optional.of(new SpecificationTypeRecord("configuration", name, declaration, parameters, List.of()));
```

Comment:
My feeling is that the inlined version is much noisier and has fewer landmarks for the reader. I personally like separating the arguments out in this case, since there are so many arguments -- the names really help in understanding what's being manipulated here.

Reply:
Very true, especially for this example.

Reply:
Thanks Jonathan & Ryan. Always open to formatting suggestions. No strong opinion on this from me, so I'll leave it as-is if that's okay.

Reply:
No problem